VsCodeExtension/Server/Core.fs

```diff
@@ -126,7 +129,9 @@ let SemanticTokenModifiers =
 let initialize(input: Input<InitializeParams>): Async<Output<InitializeResult>> = async {
     if (not MSBuildLocator.IsRegistered) then MSBuildLocator.RegisterDefaults() |> ignore
     let workspace = MSBuildWorkspace.Create()
-    let! _ = (workspace.OpenSolutionAsync(input.Parameters.RootPath.Value) |> Async.AwaitTask)
+    let slnPaths = System.IO.Directory.GetFiles(input.Parameters.RootPath.Value, "*.sln")
+    let sln = slnPaths |> Seq.head
+    let! _ = (workspace.OpenSolutionAsync(sln) |> Async.AwaitTask)
```

Comment:
Can this just be `do! workspace.OpenSolutionAsync(sln) |> Async.AwaitTask`?

Reply:
I get an error about
`This expression was expected to have type
    'Solution'    
but here has type
    'unit' `

Reply:
Maybe it'd have to be `do! (workspace.OpenSolutionAsync(sln) |> Async.AwaitTask)`. Probably not worth anymore consideration right now.